### DISCUSSION TITLE
extract patterns out the the negated entities

### DISCUSSION THREAD

**Discussion by hollywzw** [explosion/spaCy#7705] [Help: Coding & Implementations] [answered]
Hi, how do we extract the patterns of the negations identified? Is there any standard methods of negex we can use?

Like if I want to know which negation pattern below was identified for the negated entity.

NegEx Patterns
psuedo_negations - phrases that are false triggers, ambiguous negations, or double negatives
preceding_negations - negation phrases that precede an entity
following_negations - negation phrases that follow an entity
termination - phrases that cut a sentence in parts, for purposes of negation detection (.e.g., "but")

**Comment by polm** [comment, marked as answer]
There's a 3rd-party plugin called [negspacy](https://github.com/jenojp/negspacy) based on negex.

**Reply by jenojp** [reply]
see https://github.com/jenojp/negspacy/issues/36
